src/kernel/kernel.c

```diff
@@ -184,3 +184,19 @@ void __attribute__((noreturn)) context_switch_finish(context prev, context next,
     ((void (*)(u64, u64))a)(arg0, arg1);
     runloop();
 }
+
+#ifndef CONFIG_TRACELOG
+void tprintf(symbol tag, tuple attrs, const char *format, ...)
+{
+    vlist a;
+    buffer b = little_stack_buffer(256);
+    vstart(a, format);
```

Comment:
I know usage is a little inconsistent in the kernel, but it's probably not a bad idea to include the matching vend() call

Reply:
Ok, this is fixed in both definitions of tprintf().